### ISSUE TITLE
DV | Contact information content

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#116050]
## Issue Description

On the Dependents Verification form, the following updates need to be made:

- After successfully updating a field (mailing address, email address, mobile phone number, or international phone number), a success alert needs to be shown at the top of the page
  <img width="500" alt="we updated your mailing address success alert" src="https://github.com/user-attachments/assets/8ac1d827-51e9-4273-a8bf-3a4c83ed5131" />
- "Mobile phone number" should be changed to "Home phone number"
- "International phone number" should be changed to "International number" from [design](https://www.figma.com/design/bvj72inycD0iZkuCbjYTWL/Dependent-Verification-MVP?node-id=9072-28479&t=zpqRVdfkOcXZjMmi-4)
  <img width="500" alt="Mobile phone number should be home phone number and international phone number should be changed to international number" src="https://github.com/user-attachments/assets/a9e5da17-04ba-4b5c-92e3-dea59cd07f77" />
- All edit pages need the form navigation buttons to include a `full-width` property

    <img width="500" alt="Update and cancel buttons are not filling their containers and have excessive space between them" src="https://github.com/user-attachments/assets/fb4c64ec-3a3e-4d91-a011-92c4f935b47e" />

---
## Tasks
- [ ] Show success alert after successfully editing a contact information field
- [ ] Change "Mobile phone number" to "Home phone number"
- [ ] Change "International phone number" to "International number"
- [ ] Add `full-width` to form navigation buttons to fix edit pages
- [ ] Add unit tests for success alert

## Acceptance Criteria1
- [ ] Veteran contact information page matches design
- [ ] Success alert shows after editing contact information
- [ ] Success alert is hidden after returning to the page
- [ ] Edit page update and cancel buttons are wider and match design
- [ ] All tests passing

**Comment by Midge-dev** [comment]
@Mottie the Edit page buttons currently have `full-width` and look fine locally for me

<img width="681" height="363" alt="Image" src="https://github.com/user-attachments/assets/ebce99b8-513b-49c3-9e8f-c5ea7ac19c8e" />

**Comment by Mottie** [comment]
Hmm, my build was acting weird earlier today. Maybe I was on an older branch while I was testing... I'll check again tomorrow

**Comment by Mottie** [comment]
Updates have been merged and are ready for validation in staging.

**Comment by fiorella-io** [comment]
@Mottie We need to update 'application' to 'form'

<img width="60%" src="https://github.com/user-attachments/assets/81cb6915-5729-4cc0-97e5-028b8bae3e08" />

**Comment by hnvt1989** [comment]
I found an instance where “international phone number” may need to be updated to “internal number”: Edit International phone number.

<img width="812" height="412" alt="Image" src="https://github.com/user-attachments/assets/5d699a61-7b59-41aa-9a92-c68e99977563" />

**Comment by Mottie** [comment]
Some changes are still needed, but we have a code freeze in place prior to staging review, so we'll work on these updates after.

**Comment by fiorella-io** [comment]
Please tag @davidakennedy if there's any design QA that needs to be reviewed after the code freeze
